packages/typed-message/tests/encoder.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/await-thenable */
```

Comment:
hmm why?

Reply:
There is a bug in this file for eslint as seen below.
<img width="904" alt="image" src="https://user-images.githubusercontent.com/3579069/165031800-1ad11194-ed76-4dd2-8888-d066796c8f1f.png">

Reply:
well, please revert this change. I have fixed it in the develop branch in another way.

Reply:
please revert this change. I have fixed it in the develop branch in another way.

Reply:
done.